lib/modules/modulehelpers.py

```diff
@@ -11,9 +12,22 @@ def get_dates(msg):
     dates = []
     for match in matches:
         dates.append(match)
-
     return dates
 
+def get_book_name_from_isbn(isbn: str):
```

Comment:
This is good, it would be great if there was a link to good docs explaining the api, just add a line here :)

Reply:
Added